RecoMET/METPUSubtraction/src/noPileUpMEtAuxFunctions.cc

```diff
@@ -18,20 +18,17 @@ int isVertexAssociated(const reco::PFCandidate& pfCandidate,
     vtxAssociationType = noPuUtils::kChNoAssoc;
     for ( PFCandToVertexAssMap::const_iterator pfCandToVertexAssociation = pfCandToVertexAssociations.begin();
```

Comment:
@slava77 working with Ptrs seems to give a factor 5 improvement :
deltaR matching : 
TimeReport             per event        per module-run      per module-visit 
TimeReport        CPU       Real        CPU       Real        CPU       Real Name
TimeReport   0.001033   0.001035   0.001033   0.001035   0.001033   0.001035 jvfJetIdForPFNoPUMEt

Ptr matching :
TimeReport             per event        per module-run      per module-visit 
TimeReport        CPU       Real        CPU       Real        CPU       Real Name
TimeReport   0.000236   0.000237   0.000236   0.000237   0.000236   0.000237 jvfJetIdForPFNoPUMEt

For the decision to run the algorithms at the RECO/miniAOD sequences, we can discuss at the next RECO/AT meeting